core/server/api/subscribers.js

```diff
@@ -192,6 +192,29 @@ subscribers = {
 
         /**
          * ### Delete Subscriber
+         * If we have an email param, check the subscriber exists
+         * @type {[type]}
+         */
+        function getSubscriberByEmail(options) {
+            if (options.email) {
+                return models.Subscriber.getByEmail(options.email)
```

Comment:
passing the model options is missing - getByEmail accepts to params (email, options). this is important for permissions

Reply:
👍 thanks, I'll push the fix. Is there a way to test that passing options like this isn't missed?

Reply:
Hmm in this specific case, as soon as you pass a transaction from outside, it would never finish.